File: src-tauri/src/commands/games.rs
use crate::games::{Game, GAMES};

#[tauri::command]
pub async fn get_games() -> &'static [Game<'static>] {
    &*GAMES
}

#[tauri::command]
pub async fn get_games_popularity() -> &'static str {
    include_str!("../gameReviews.json")
}

#[tauri::command]
pub async fn get_game_mods_downloads() -> &'static str {
    include_str!("../gameModDownloads.json")
}

File: scripts/game-updater/src/games.ts
import { projectRootDir } from "./paths.ts";

import { Game } from "../../../src/types.d.ts";

import games from "../../../src-tauri/src/games/games.json" with { type: "json" };
export const gamesJsonPath = projectRootDir + "/src-tauri/src/games/games.json";

export default games as Game[];